other/containers/tilck-ci-build/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Also tried to use debian/stable-slim as base, but image size went up compared
```

Comment:
Please add the `# version: v1` comment here at line 1.

Reply:
Yeah, sorry for that. I explained what i misunderstood in the other comment referencing this.